src/coredump/coredump.c

```diff
@@ -857,9 +860,9 @@ static int submit_coredump(
         }
 
         if (mount_tree_fd >= 0 && arg_access_container) {
-                r = setup_container_mount_tree(mount_tree_fd, &root);
+                r = attach_mount_tree(mount_tree_fd);
                 if (r < 0)
-                        log_warning_errno(r, "Failed to setup container mount tree, ignoring: %m");
+                        root = NULL;
         }
```

Comment:
hmm, with this `root` will be set to the wrong root in case arg_access_container is false. this doesn't look right?

Reply:
if arg_access_container is false then mount_tree_fd is invalid, i.e. we don't even send it, so I guess we could just check for fd validity.

Reply:
I guess I could just set root to "/" if attaching alternative mount tree fails, that would work as well because in parse_core we check or "/" and don't call dwfl_set_sysroot() in such case.